devmode-test.adoc

```diff
@@ -1,3 +1,3 @@
 === Running the tests
 
-Because you started Open Liberty in dev mode, press the `enter/return` key to run the tests.
+Because you started Open Liberty in dev mode, press the `enter/return` key from the command-line session that you started the dev mode to run the tests.
```

Comment:
Because you started Open Liberty in dev mode, you can run the tests by pressing the `enter/return` key from the command-line session where you started dev mode.